### ISSUE TITLE
Formally Create a Transition Plan from Slack to Rocket.Chat

### ISSUE THREAD

**Issue by AaronBurrSir** [calpirateparty/CAPP#4]
Right now, the majority of our userbase is using Slack.

We need to figure out a plan of how best to transition CAPP as a whole to Rocket.Chat

Depends on https://github.com/calpirateparty/GLAP/issues/9 and https://github.com/calpirateparty/GLAP/issues/16 being completed first. 

**Other Requirements:** (these can be setup as new issues, listing them out for now.)
- Documentation about how to create a Rocket.Chat account. This needs to be dead simple so your grandmother could do it. 
- Documentation on which Mobile Apps work with Rocket.Chat (for iOS and Android) and how to connect to the chat with them. This needs to be dead simple so your grandmother could do it. 
- After documentation has been completed, do a messaging campaign announcing that Slack will be closed and that all real-time communication will be transitioning to Rocket.Chat. Throw this out on all social media channels including but not limited to Facebook, Instagram, Twitter and email. 
- Setup a timeline of how long to message the change, when to make Slack read only and when to shut it down for good. 
 - My suggestion is 2 weeks for messaging followed by read-only access for a month followed by complete shuttering of the Slack server after that.

**Comment by sptankard** [comment]
Should also reference calpirateparty/GLAP#14, particularly re: your 1st and 2nd bullet points.
(Documentation about how to create a Rocket.Chat account, and which Mobile Apps to use)
 
https://github.com/calpirateparty/GLAP/issues/14#issuecomment-297248573

**Comment by 2041876935** [comment]
Personally, I've just been waiting for the e-mail backend, then I was just gonna do it. Shutter Slack, condense Rocket.Chat channels, and make appropriate announcements.... If this less formal approach is cool with you guys, lets **CLOSE** this yeah?

**Comment by AaronBurrSir** [comment]
@2041876935 Let's not do it so fast. We need to give our users time to acclimate to the change. Doing it immediately *before* announcements have been made is a **very very bad** idea.

**Comment by AaronBurrSir** [comment]
@2041876935 Let's get email running and the Rocket.Chat channels condensed before closing Slack or anything. Also we shouldn't *shutter* slack completely but make it read only instead.

**Comment by sptankard** [comment]
I agree with @AaronBurrSir, we need to ensure that we don't lose members in the transition.

**Comment by sptankard** [comment]
Slack to rocketchat transition is at this point... mostly... complete. I think we still need to pull in some stragglers/make sure we didn't lose anybody.
